src/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -16,24 +16,66 @@ public void GetAccessControl_DirectoryInfo_InvalidArguments()
         }
 
         [Fact]
+        public void GetAccessControl_DirectoryInfo()
```

Comment:
Nit: consider appending _ReturnsValidObject or something like that to the name of the test